docs/operate/reference/components/camera/webcam.md

```diff
@@ -151,6 +151,10 @@ system_profiler SPCameraDataType
 
 The Unique ID displayed for each camera is the `video_path`.
 
+If you are using MacOS version 14.0.0 Sonoma or later, you will need to give viam-server permissions to access webcams.
+After running viam-server for the first time, check for a pop-up message asking for Camera permissions.
```

Comment:
```suggestion
After running `viam-server` for the first time, check for a pop-up message asking for camera permissions.
```
Would this actually be
```suggestion
When you run `viam-server` for the first time, a pop-up message will ask for camera access.
Click **OK**.
```
or what does the message actually say and when does it appear? I'm running sonoma but don't see viam-server listed in my camera permissions.

Reply:
Thanks for testing on Sanoma! It appears that the permission change actually happened in Sequoia
Found this helpful link [here](https://cascable.se/help/macos-sequoia-permissions/).

Reply:
Ah thanks for the link. So, if you click Allow I'd think you'd not *also* need to go to permissions, would you?

Reply:
Yes, correct. Going to System Settings would just be to confirm that permissions are allowed for viam-server